src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -239,13 +239,13 @@ export default class OrgPerson extends Mixins(EntityFilterMixin, CommonMixin) {
   private created (): void {
     if (this.initialValue) {
       this.orgPerson = { ...this.initialValue }
-      this.orgPerson.person = { ...this.initialValue.person }
+      this.orgPerson.officer = { ...this.initialValue.officer }
```

Comment:
Maybe should change 'orgPerson' to 'orgOfficer'? Leaving this one up to you.

Reply:
Thanks